src/resolvers.ts

```diff
@@ -491,6 +489,12 @@ export function getModuleResolver(
   };
 }
 
+export function resolveBuiltin(specifier: string): string {
+  if (builtins.has(specifier)) return builtins.get(specifier)!;
+  if (!specifier.startsWith("observablehq:")) throw new Error(`not built-in: ${specifier}`);
+  return `/_observablehq/${specifier.slice("observablehq:".length)}${extname(specifier) ? "" : ".js"}`;
```

Comment:
took me a moment to figure out what case that was!

one example: `import {mutable} from "observablehq:stdlib/mutable.js";`

Reply:
Yep! The primary one is `observablehq:stdlib/inputs.css`.